create-base.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -eo pipefail
+if [ ! "${FREEBSD_VERSION}" ]; then
```

Comment:
Is this envar already declared or loaded from somewhere?

Reply:
It was originally provided by sourcing `./check-env.sh`. However, that would return "14.0" rather than "23.11" currently for CheriBSD. To avoid this snag, I'm using the manifests themselves to provide the appropriate version numbers. This option seemed more robust than trying to hardcode the date-based versions, like "23.11", used for CheriBSD.